From: Jan Beich <jbeich@FreeBSD.org>
Date: Sat, 7 Aug 2021 05:35:55 +0000
Subject: meson: explicitly set C99 standard after 57fb8074161c

GCC < 5 uses C89 by default, so the build failed with:

src/plugins/cairo/libdecor-cairo.c: In function 'ensure_cursor_theme':
src/plugins/cairo/libdecor-cairo.c:2124: error: 'for' loop initial declaration used outside C99 mode

Origin: upstream, 0.1.1, commit:6653451d6cefe99c551ed9962112fc4d61cd04c2
---
 meson.build | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 94c7308..ad4ccb9 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,10 @@
 project('libdecor', 'c',
   version: '0.1.0',
   meson_version: '>= 0.47.0',
-  license: 'MIT'
+  license: 'MIT',
+  default_options: [
+    'c_std=c99',
+  ],
 )
 
 version_array = meson.project_version().split('.')
